runtime.getURL fixes (#35136)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/runtime/geturl/index.md

### Return value

> `string`. The fully-qualified URL to the resource. For example, `moz-extension://e15cdd35-f9b7-4786-a39d-7d801bd2709e/help/open-in-container.html`.

Reviewer: Let's drop this example. It is not obvious from just this description how the URL is derived from the input.

Furthermore, the code example below already offers a comment showing the format of the return value in Firefox specifically.

```suggestion
`string`. The fully-qualified URL to the resource within the extension.
```
